api/apps/articles/model.coffee

```diff
@@ -21,13 +21,15 @@ request = require 'superagent'
 schema = (->
   author_id: @objectId().required()
   tier: @number().default(2)
-  slug: @string().allow(null)
+  slug: @string().allow('', null)
+  slugs: @array().items(@string()).allow('',null,[])
```

Comment:
We shouldn't be allowing the client to change `slugs`. B/c this retains history I'd rather this be protected behind the API. (e.g. Orta might want make a native Writer app and we don't want him abusing this power :smiley_cat: )